recipe/meta.yaml

```diff
@@ -26,6 +25,7 @@ requirements:
   host:
     - cudnn
     - libcublas-dev     # [(cuda_compiler_version or "").startswith("12")]
+    - cuda-driver-dev     # [(cuda_compiler_version or "").startswith("12")] 
```

Comment:
This seems to be correct to me, if you need the header and to link to `libcuda.so`.